ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -9,8 +9,7 @@ workflow:
     - chain: ipi-openstack-post
     env:
       OPENSTACK_EXTERNAL_NETWORK: "external"
-      OS_CLOUD: "openstack-cloud"
-      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
+      OS_CLOUD: "openstack"
```

Comment:
There are a few more places that are still defaulting to "openshift-cloud" for the value of the "OS_CLOUD" environment variable. Should they be changed, too?
https://github.com/openshift/release/blob/2866cc0b21c185258ddc684ef3fddc889bbfdfb4/ci-operator/step-registry/openshift/e2e/openstack/csi/openshift-e2e-openstack-csi-workflow.yaml#L12
https://github.com/openshift/release/blob/2866cc0b21c185258ddc684ef3fddc889bbfdfb4/ci-operator/step-registry/openshift/e2e/openstack/minimal/openshift-e2e-openstack-minimal-workflow.yaml#L10
https://github.com/openshift/release/blob/2866cc0b21c185258ddc684ef3fddc889bbfdfb4/ci-operator/step-registry/openshift/upgrade/openstack/loki/openshift-upgrade-openstack-loki-workflow.yaml#L20

Reply:
Actually is a small change in each file. I'll include the files in here for that.